librz/bin/format/elf/elf.c

```diff
@@ -115,11 +115,11 @@ typedef struct reginfo {
 } reginfo_t;
 
 static reginfo_t reginf[ARCH_LEN] = {
-	{ 160, 0x5c },
-	{ 216, 0x84 },
-	{ 72, 0x5c },
-	{ 272, 0x84 },
-	{ 272, 0x84 }
+	{ 160, 0x48 },
```

Comment:
Where from this change comes?

Reply:
Before, these offsets were from the very start of the note, now they are from the start of the inner payload since that is where `parse_note_prstatus()` starts to read and it also makes more sense.

More specifically, these notes are:
3x32bit + a string of size defined in one of these 32bit values + optional padding + payload.
In the case of `NT_PRSTATUS` ones, the string is `CORE\0` (usually, not sure if always guaranteed to be this. The previous implementation relied on this length to be always the same, now it does not anymore), so you get
3*4 + 5 + 3 padding = 0x14 which is exactly the difference here.

Reply:
Could you please add this as a comment?

Reply:
This by itself doesn't make much sense as a comment because it primarily explains why it's different to before, but I will add something about the meaning of these values.

Reply:
Yes, that's what I meant.

Reply:
added